### PR TITLE
Rename heterogeneous to hybrid

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

**Review by QuanluZhang — APPROVED** [review]

**Comment by QuanluZhang** [comment]
please fix pipeline error, my pr will merge your changes after this pr is merged

**Review by J-shang — APPROVED** [review]

File: examples/trials/mnist-tfv1/config_heterogeneous.yml
Comment (J-shang) [inline examples/trials/mnist-tfv1/config_heterogeneous.yml:6]: `config_heterogeneous.yml ` -> `config_hybrid.yml `

Comment (QuanluZhang) [reply examples/trials/mnist-tfv1/config_heterogeneous.yml:6]: let me fix this one in my pr :)

**Pull request merged**
